notebooks/ir/sparse/bm25_retrieval.ipynb

```diff
@@ -325,11 +325,9 @@
   }
  ],
  "metadata": {
-  "interpreter": {
-   "hash": "a9fac2af06ead429517f1008e4b4209682dcd5493b65bc6429a3a9c2b98533d6"
-  },
   "kernelspec": {
-   "display_name": "Python 3.8.12 64-bit ('primeqa': conda)",
+   "display_name": "primeqa-kpfadnis",
```

Comment:
can we not have user names here? Thanks.

Reply:
@jdpsen The `display_name` get set automatically as I open and run notebook in VSCode. I can try to configure my vscode for future but can you rename in it `Tapex` PR?